server/routes/shared/riskPresenter.ts

```diff
@@ -23,10 +24,43 @@ export default class RiskPresenter {
     riskImminence: this.riskSummary?.summary.riskImminence,
   }
 
+  get lastUpdated(): string {
+    const assessedOn = this.riskSummary?.assessedOn
+      ? DateUtils.formattedDate(this.riskSummary.assessedOn, { month: 'short' })
+      : 'assessment date not found'
+
+    return `Last updated: ${assessedOn}`
+  }
+
   get roshAnalysisHeaders(): string[] {
     return ['Risk to', 'Risk in community']
   }
 
+  private get overallRoshScore(): string {
+    if (!this.riskSummary) {
+      return 'Rosh score not found'
+    }
+
+    const { riskInCommunity } = this.riskSummary.summary
+
+    const roshRankings = ['VERY_HIGH', 'HIGH', 'MEDIUM', 'LOW']
+
+    let i = 0
+
+    // For each roshRanking, check to see if it is defined. If not, step to the
+    // next, rinse and repeat.
+    while (riskInCommunity[roshRankings[i]] === undefined) {
+      i += 1
```

Comment:
Whilst we have a contract defined by ARN and there is an expectation that they should never violate it, if they do violate it then it shouldn't break our app.

I think here that there would be an infinite recursion on contract violation - for instance if they return something like `VERY_LOW`.